src/types/Errors.ts

```diff
@@ -0,0 +1,7 @@
+export enum Errors {
+  loadError = 'Unable to load todos',
+  emptyTitle = 'Title should not be empty',
+  addTodoError = 'Unable to add a todo',
+  deleteTodoError = 'Unable to delete a todo',
+  updateTodoError = 'Unable to update a todo',
```

Comment:
great solution for error messages!)

please just update the fisrt letter to UpperCase like
`loadError` to `LoadError`